hack/release/create.sh

```diff
@@ -17,18 +17,18 @@
 # builds binaries between the commits
 # Use like: create.sh <release-version> <next-prerelease-version>
 # EG: create.sh v0.3.0 v0.4.0-alpha
-set -o nounset
-set -o errexit
-set -o pipefail
+set -o errexit -o nounset -o pipefail
 
+# cd to the repo root
```

Comment:
why do you move this before checking?
if there are no arguments and the next `if` fails we don't need to cd to the repo root, right?

Reply:
for consistency with the other scripts. it doesn't matter but it's easier to spot check these between the scripts (EG if we tweak the `REPO_ROOT` logic)